src/platform/PlatformFunctions.ts

```diff
@@ -56,6 +71,53 @@ class PlatformFunctions {
   ): Promise<RequestData | BufferedFile> {
     throw new Error('tryBufferData not implemented.');
   }
+
+  /**
+   * Creates an HTTP client which uses the Node `http` and `https` packages
+   * to issue requests.
+   */
+  createNodeHttpClient(agent?: http.Agent | null): typeof HttpClient {
```

Comment:
I'm surprised cloudflare doesn't complain about this use of Node `http.Agent`

Reply:
The type / import goes away when transpiled to JS